.travis.yml

```diff
@@ -1,6 +1,8 @@
+language: cpp
 
-env:
-  - CXX=g++
+compiler:
+  - clang
+  - gcc
```

Comment:
The compiler section is only used to define the build matrix. When the build starts you can still set CXX to clang++ or g++
Doc here: http://docs.travis-ci.com/user/languages/cpp/#Choosing-compilers-to-test-against